### ISSUE TITLE
Max/min rows aren't being handled correctly when submitting repeater (GQL and built-in)

### ISSUE THREAD

**Issue by tremby** [verbb/formie#876]
**Description**

I'm trying to submit a repeater via GQL. My repeater doesn't have any min or max number of rows set, but I'm getting an error regarding limits when I submit one or more rows. There is no error, and the submission is accepted, if I send `myRepeater: { rows: [] }`.

**Steps to reproduce**
1. Make a form with a repeater, leaving min and max instances blank
2. Attempt to submit a form via GQL, with at least one row being sent for the repeater, such as `myRepeater: { rows: [{ myRepeaterField: "val" }] }`.
3. See error (`my repeater must have one item.`).

### Form settings
Please pick from the options below.

- Multi-page form: n/a
- Submission Method: GQL
- Client-side Validation: n/a
- Custom Form Templates: n/a

**Additional info**
- Plugin version: dev-craft-4 4ddb6e3
- Craft version: 4.1.x-dev at 56878d94
- Multi-site: Yes

**Additional context**

**Comment by ljclark** [comment]
Also getting this issue on a normal Twig based implementation - doesn't allow the submission to submit., 

It's throwing the same error `xxx repeater must have one item.` despite being populated, and there being no minimum's set. Same result on repeaters that do have min or max's set aswell. 

### Form settings
Please pick from the options below.

Multi-page form: Yes
Submission Method: Ajax
Client-side Validation: Yes
Custom Form Templates: No
Additional info

Plugin version: 2.0.0-beta.6
Craft version: Craft Pro 4.0.1
Multi-site: No
Additional context

This is a very vanilla install, no other plugins other than Formie are used.

--

The logs look to be creating the repeater rows successfully too, but it's still failing, so not sure what's going on:

```log
2022-05-11 14:50:17 [INFO] Submission triggered for formHandleHere.
2022-05-11 14:50:17 [ERROR] Couldn’t save submission due to errors - {"repeaterFieldTest":["repeaterFieldTest must have one item."]}.
2022-05-11 14:50:17 [INFO] Request context:
$_GET = []

$_POST = [
    'CRAFT_CSRF_TOKEN' => 'xxx123'
    'action' => 'formie/submissions/submit'
    'handle' => 'formHandleHere'
    'siteId' => '1'
    'fields' => [
        'repeaterFieldTest' => [
            'rows' => [
                'new1' => [
                    'fields' => [
                        'innerField' => 'field value 1 example'
                    ]
                ]
                'new2' => [
                    'fields' => [
                        'innerField' => 'field value 2 example'
                    ]
                ]
            ]
            'sortOrder' => [
                0 => 'new1'
                1 => 'new2'
            ]
        ]
    ]
]
```

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.6",
  "...": "..."
}
```

Then run `composer update`.

**Comment by ljclark** [comment]
Awesome, thanks @engram-design - that's sorted it for me 🎉

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.7](https://github.com/verbb/formie/releases/tag/2.0.0-beta.7)
